src/read.c

```diff
@@ -987,7 +987,7 @@ static avifResult avifDecoderFindMetadata(avifDecoder * decoder, avifMeta * meta
 
         if (!decoder->ignoreExif && !memcmp(item->type, "Exif", 4)) {
             avifROData exifContents;
-            avifResult readResult = avifDecoderReadItem(decoder, item, &exifContents, 0);
+            avifResult readResult = avifDecoderReadItem(decoder->io, item, &exifContents, 0);
```

Comment:
This change makes it obvious at a glance that avifDecoderReadItem doesn't modify any fields in 'decoder'.